cysync/src/store/hooks/flows/useDeviceAuth.ts

```diff
@@ -105,7 +105,12 @@ export const useDeviceAuth: UseDeviceAuth = isInitial => {
       process.env.BUILD_TYPE === 'debug' &&
       localStorage.getItem('disableProvision') === 'true';
 
-    setConfirmed(1);
+    if (inTestApp) {
+      setConfirmed(2);
+      setVerified(1);
+    } else {
+      setConfirmed(1);
+    }
```

Comment:
I think this already happens from the `protocol` module

Reply:
It wasn't happening, So I had to. Could you recheck?